internal/pkg/deploy/cloudformation/stack/lb_web_svc.go

```diff
@@ -103,8 +111,28 @@ func (s *LoadBalancedWebService) Template() (string, error) {
 	return content.String(), nil
 }
 
+func (s *LoadBalancedWebService) loadBalancerTarget() (*string, *string) {
+	containerName := s.name
+	containerPort := strconv.FormatUint(uint64(aws.Uint16Value(s.manifest.Image.Port)), 10)
+	// Route load balancer traffic to main container by default.
+	targetContainer := aws.String(containerName)
+	targetPort := aws.String(containerPort)
+	mftTargetContainer := s.manifest.TargetContainer
```

Comment:
The `TargetContainer` made me think that we probably want to make a change to `"svc show"`. If a target container is set, then the Service Discovery endpoint should display the port of the sidecar.

https://github.com/aws/amazon-ecs-cli-v2/blob/ce5d2d9dc01c82b96a10e5e3e9a94d2b468eef40/internal/pkg/describe/lb_web_service.go#L158

Not saying it should be part of this PR but maybe we can create an issue? Maybe we could create a parameter like `ServiceDiscoveryPort` in our stacks, that can be used across all services so that it'd also work for a backend service.

Reply:
yeah i'll do that

Reply:
done!